docs/configure_open_edx.md

```diff
@@ -71,6 +94,7 @@ In Open edX (derived from instructions [here](https://edx.readthedocs.io/project
   - Select the default example site
   - The slug field **MUST** match the `Backend.name`, which for us is `
 mitxpro-oauth2`
+  - Client Id should be the client id from the xPro Django Oauth Toolkit Application
```

Comment:
Client secret key too?  Not sure, I added both just in case.

Reply:
No, that should go in `lms.auth.json`